scripts/netContain/Dockerfile

```diff
@@ -22,7 +22,7 @@ FROM ubuntu:16.04
 # ENV export https_proxy=https://proxy.localhost.com:8080
 
 RUN apt-get update \
- && apt-get install -y openvswitch-switch=2.5.0-0ubuntu1 \
+ && apt-get install -y openvswitch-switch=2.5* \
```

Comment:
1: Do we need to change version in our ansible repo as well ? 
2: As we have not started supporting ubuntu officially, do we need to add this specific to ubuntu ? 
3: Some time back, we had issues with ovs repo, so we changed to openstack kilo repo to install ovs.
https://github.com/contiv/ansible/blob/master/roles/contiv_network/tasks/ovs.yml#L3
Do we need to follow same for ubuntu as well ?